pkg/clusterapi/controllermanagerargs.go

```diff
@@ -0,0 +1,10 @@
+package clusterapi
+
+import (
+	"github.com/aws/eks-anywhere/pkg/cluster"
+)
+
+func SetControllerManagerArgs(clusterSpec *cluster.Spec) ExtraArgs {
```

Comment:
Why do we need a separate function and file for this? Do we anticipate this function to be exported by other packages in the future? I would argue that this is mainly a helper function in `apibuilder.go`

Reply:
These will eventually be used for all other providers as well.